io.github.java_decompiler.jd-gui.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: io.github.java_decompiler.jd-gui
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: jd-gui
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=host:rw

```

Comment:
that's a very broad filesystem permission

Reply:
It's a Java App which doesn't support portals

Reply:
and? You git it full host access?

Reply:
The User should be able to open a File anywhere n the System

Reply:
no

Reply:
If this is not possible the App is broken